Server/src/server.cpp

```diff
@@ -455,29 +449,25 @@ int Server::handleLsCommand(char *command) {
         std::string directoryContents = "Directory listing for " + cwd + "\n";
 
         // directory_iterator is used to traverse all the entries of directory
-        for (const auto & entry : std::filesystem::directory_iterator(cwd)) {
+        for (const auto& entry : std::filesystem::directory_iterator(cwd)) {
             directoryContents += entry.path().filename().string() + "\n";
         }
 
-        directoryContents += '\f';
```

Comment:
Why did you remove this?

Reply:
Also the lines below that line

Reply:
this is the point of double \f.
below, just because everywhere else it was used handleSend command, but in this place - winsock send.

Reply:
Alright, I was probably too tired yesterday to actually look at the code lmao